src/applications/vaos/newAppointmentFlow.js

```diff
@@ -30,26 +41,27 @@ export default {
   },
   typeOfCare: {
     url: '/new-appointment',
-    next(state) {
-      if (getFormData(state).typeOfCareId === SLEEP_CARE) {
-        return 'typeOfSleepCare';
+    async next(state) {
+      let nextState = 'vaFacility';
+
+      if (isSleepCare(state)) {
+        nextState = 'typeOfSleepCare';
+      } else if (isCommunityCare(state)) {
+        try {
+          const data = await getCommunityCare(
+            '/vaos/community-care/eligibility',
```

Comment:
LGTM.  Just curious where this `'/vaos/community-care/eligibility'` is coming from? I know it is just a mock API, but `getCommunityCare()` doesn't accept any parameters.

Reply:
It was in the original code comment.

    // async next(state) {
    //   try {
    //     const data = await apiRequest('/vaos/community-care/eligibility');
    //
    //     if (isAllowedTypeOfCare(state) && isEligible(data.eligibility)) {
    //       return 'typeOfFacility';
    //     }
    //
    //     return 'vaLocation';
    //   } catch (e) {
    //     return 'vaLocation';
    //   }
    // },

I kept it for reference since I don't know what the endpoint is going to look like.

Reply:
Looks like I need one more reviewer to approve these changes.